README.md

```diff
@@ -87,3 +87,16 @@ additional properties not defined in the schema are passed in through the reques
 * For new pages, create a new route in `client/src/routes.js` and import a page component from `client/src/pages`
 * Refrain from making the Redux state too big as it will affect speed performance. React’s Context API is ONLY helpful for avoiding nested prop threading so stick with Redux where Context fails.
 
+## Deployment
+
+### Review branches
+
+Every time you push code up to the repository, a build based off of your feature review branch will be deployed to AWS. Note that in order for a build to run, the branch name must have the `feature/` prefix. You can view the build logs in [GitHub Actions](https://github.com/FightPandemics/FightPandemics/actions). After the build successfully completes, you can access your built app at `<branch_name>.fightpandemics.xyz`, where `<branch_name>` is the *sanitized* branch name of your feature branch (i.e. if the branch name has a `/` in it, it will be converted to a `-`, e.g. `feature/hot-potato` will be converted to `feature-hot-potato`). Note that it may take a few minutes for the app to be accessible, or for changes to be reflected, since it takes time for AWS to spin up the Docker containers.
```

Comment:
ok good to know, too bad we can't have other branch names deployed like  `/bugfix` like this though
also wonder if it's automatically removed when the branch is deployed?

Reply:
Actually, I can add `bugfix/`, as well as any other prefix we may want to trigger builds.

And yes, the review build would be torn down after the branch is merged to master, or if the branch PR is closed without merging.